### DISCUSSION TITLE
collections as search query in beatmap search

### DISCUSSION THREAD

**Discussion by Arnacino** [ppy/osu#25422] [Ideas]
Hi everyone, 
since i have many collections and some of them have many beatmaps in common, it would be pretty cool to be able to do a search query to look for maps that are in multiple collections.
something like "collection=Alternate, Bangers" would have as a result all the beatmaps that are contained in both the Alternate and the Bangers collections.

**Comment by Orofil** [comment]
I think there should similarly be a feature to exclude a collection from the results.
For example: `-collection="Collection name"` would show all the maps that aren't in that collection.
